mindsdb/integrations/handlers/openbb_handler/openbb_tables.py

```diff
@@ -43,6 +44,20 @@ def _get_params_from_conditions(self, conditions: List) -> Dict:
 
             return params
 
+        def _select_dataframe(self, query: ast.Select, df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
looks like this function could be used instead: mindsdb.api.mysql.mysql_proxy.utilities.sql.query_df